lib/src/ui.dart

```diff
@@ -10,29 +10,107 @@ import 'network_event.dart';
 import 'network_logger.dart';
 
 /// Overlay for [NetworkLoggerButton].
-class NetworkLoggerOverlay extends StatelessWidget {
-  NetworkLoggerOverlay._({Key? key}) : super(key: key);
+class NetworkLoggerOverlay extends StatefulWidget {
+  NetworkLoggerOverlay._({this.right, this.bottom, Key? key}) : super(key: key);
+
+  double? bottom;
+  double? right;
 
   /// Attach overlay to specified [context].
   static OverlayEntry attachTo(
     BuildContext context, {
     bool rootOverlay = true,
+
+    /// Initial distance from [NetworkLoggerButton] to bottom edge of screen
+    double? bottom,
+
+    /// Initial distance from [NetworkLoggerButton] to right edge of screen
+    double? right,
   }) {
     // create overlay entry
-    var entry = OverlayEntry(
-      builder: (context) => NetworkLoggerOverlay._(),
+    final entry = OverlayEntry(
+      builder: (context) => NetworkLoggerOverlay._(bottom: bottom, right: right),
     );
     // insert on next frame
     Future.delayed(Duration.zero, () {
-      Overlay.of(context, rootOverlay: rootOverlay)?.insert(entry);
+      final overlay = Overlay.of(context, rootOverlay: rootOverlay);
+
+      if (overlay == null) {
+        throw Exception(
+          'FlutterNetworkLogger:  No Overlay widget found. '
+          '                       The most common way to add an Overlay to an application is to include a MaterialApp or Navigator above widget that calls NetworkLoggerOverlay.attachTo()',
+        );
+      }
+
+      overlay.insert(entry);
     });
     // return
     return entry;
   }
 
+  @override
+  State<NetworkLoggerOverlay> createState() => _NetworkLoggerOverlayState();
+}
+
+class _NetworkLoggerOverlayState extends State<NetworkLoggerOverlay> {
+  late double bottom = widget.bottom ?? 30;
+  late double right = widget.right ?? 30;
```

Comment:
```suggestion
  double bottom = widget.bottom;
  double right = widget.right;
```

Reply:
I'm not sure about this one tho. Do we need late to set value from widget or it's not a necessity?